cmd/image-builder/main_test.go

```diff
@@ -559,6 +559,39 @@ func Test_appendMissing(t *testing.T) {
 	}
 }
 
+func Test_parseTagsFromEnv(t *testing.T) {
```

Comment:
This test looks uncompleted. It doesn't call tested function and cover only one scenario.

Reply:
Fixed